server/controllers/chartsController/helpers.js

```diff
@@ -303,3 +331,84 @@ export const calculateSubjectVariableDayCount = (
 
   return subjectsDayDataAndVariable.length
 }
+
+export const addSubjectsToResultsFromFiltersMap = ({
+  subjectsInFiltersA,
+  subjectsInFiltersB,
+  mapForResultsFromFilters,
```

Comment:
should we be more specific with these parameter names? I wonder if we want to have something like `inclusionExclusionFilteredSubjects` and `sexAtBirthFilteredSubjects` or something like that. Also, do we need to pass a map to this or could this function initialize a map and return it?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/280/commits/c30a7d514c931ad36f83916c440ff7812449f315